ironfish-cli/src/commands/wallet/mint.ts

```diff
@@ -186,10 +188,20 @@ export class Mint extends IronfishCommand {
 
     let assetData
     if (assetId) {
-      assetData = (await client.chain.getAsset({ id: assetId })).content
-      const isAssetOwner = this.isAssetOwner(assetData, accountPublicKey)
-      if (!isAssetOwner) {
-        this.error(`The account '${account}' does not own this asset.`)
+      try {
```

Comment:
Feel like maybe this `client.chain.getAsset({ id: assetId })` can be under a branch where `isMintingNewAsset` is `false`. In that case we prob would want to throw this not found error because you're trying to mint an existing asset that doesn't exist. Currently I don't think this command handles that case